Makefile

```diff
@@ -23,6 +24,7 @@ install-aur: aur
 
 install: install-aur
 	@install -Dm755 lib/aur-* -t '$(DESTDIR)$(LIBDIR)/$(PROGNM)'
+	@install -dm755 aurutils  -t '$(DESTDIR)$(ETCDIR)'
```

Comment:
`install: target directory not allowed when installing a directory`

Must be `@install -dm755 '$(DESTDIR)$(ETCDIR)/aurutils'` it seems